routes/web.php

```diff
@@ -72,27 +73,20 @@
 Route::get('/viewCart', 'PageController@viewCart')->name('viewCart');
 Route::get('/addToCart/{id}', 'PageController@addToCart')->name('addToCart');
 Route::post('/upDateCart/{id}', 'PageController@upDateCart')->name('upDateCart');
-
 Route::get('/delItemCart/{id}', 'PageController@delItemCart')->name('delItemCart');
 Route::get('/searchproduct', 'PageController@searchProduct')->name('searchProduct');
 Route::get('/logout', 'PageController@postLogout')->name('Logout');
 Route::post('/comments/{id}', 'PageController@comments');
 Route::get('/comments/{id}' , 'PageController@getComment');
-
-
+Route::get('/redirect/{social}', 'SocialAuthController@redirect')->name('Login.github');
+Route::get('/callback/{social}', 'SocialAuthController@callback');
```

Comment:
Route Socialite e chỉ cần 2 route cho toàn bộ các mạng xã hội
A đang thấy mỗi mạng xã hội của e cần 2 routes. Nếu login 10 mạng xh thì cần 20 routes à e?